### PR TITLE
Add removeItemsSelected props and hide dropdown icon to do not toggle & callback function to know if is editable or no

### PR DESCRIPTION
**Pull request by MinhOmega**
Bottom border color prop & option of no remove items selected
Hide dropdown icon to do not toggle & callback function to know if is editable or no

### CODE REVIEW

**Comment by hnthangit** [comment]
Change PR title (more detail pls) @MinhOmega

File: lib/index.js [hunk 1]
```diff
@@ -140,6 +146,7 @@ function SelectBox({
     arrowIconColor = Colors.primary,
     searchIconColor = Colors.primary,
     toggleIconColor = Colors.primary,
+    bottomBarColor = '#ddd',
```
Comment (hnthangit) [inline lib/index.js:149]: Line 148: using `Colors.primary`, why using `#ddd` in line 149

File: lib/index.js [hunk 2]
```diff
@@ -188,7 +195,7 @@ function SelectBox({
   const kContainerStyle = {
     flexDirection: 'row',
     width: '100%',
-    borderColor: '#ddd',
+    borderColor: bottomBarColor,
```
Comment (hnthangit) [inline lib/index.js:198]: bottomBarColor maybe variable, try constant because this is color

**Comment by hnthangit** [comment]
Separate branch dev and master, create feat from dev and merge into dev branch (current dev into master) @MinhOmega

File: lib/index.js [hunk 3]
```diff
@@ -112,14 +114,17 @@ function SelectBox({
     const kMultiOptionsLabelStyle = {
       fontSize: 10,
       color: '#fff',
```
Comment (hnthangit) [inline lib/index.js:116]: Color here and padding margin, can be constant?

File: lib/index.js [hunk 4]
```diff
@@ -112,14 +114,17 @@ function SelectBox({
...
+      padding: 4,
+      marginRight: 5,
       ...multiOptionsLabelStyle,
     }
     return (
       <View style={kMultiOptionContainerStyle}>
         <Text style={kMultiOptionsLabelStyle}>{label.item}</Text>
```
Comment (hnthangit) [inline lib/index.js:123]: In `kMultiOptionsLabelStyle`, what is 'k'?

**Comment by MinhOmega** [comment]
> Change PR title (more detail pls) @MinhOmega

done !!

**Comment by MinhOmega** [comment]
> Separate branch dev and master, create feat from dev and merge into dev branch (current dev into master) @MinhOmega

done. Pls re-check and merge

File: lib/index.js [hunk 5]
```diff
@@ -303,6 +312,7 @@ function SelectBox({
   }
 
   function onPressShowOptions() {
+    editStatus ? editStatus(showOptions) : null
     return () => setShowOptions(!showOptions)
   }
 }
```
Comment (hnthangit) [inline lib/index.js:318]: Line 319: using constant file or env file

**Pull request merged**